python/Ganga/test/GPI/LCG/TestLCG.py

```diff
@@ -10,14 +10,14 @@
 
 
 @external
-def test_job_complete(gpi):
+def test_job_submit(gpi):
     from Ganga.GPI import Job, LCG
 
     j = Job()
     j.backend = LCG()
     j.submit()
-    assert run_until_completed(j, timeout=1200, sleep_period=10), 'Timeout on job submission: job is still not finished'
 
+    assert j.status != 'new'
```

Comment:
I feel like it would be worth at least testing a single pass of the monitoring to make sure that it can be done. Something like:
```python
stripProxy(j).backend.master_updateMonitoringInformation([stripProxy(j)])
```
and rename the test to `test_job_monitoring`.